task_manager/models/task.py

```diff
@@ -16,3 +16,9 @@ class Task(db.Model):
 
     def __repr__(self):
         return '<Task {} {}>'.format(self.title, self.priority)
+
+
+class TaskSchema(ma.SQLAlchemyAutoSchema):
+    class Meta:
```

Comment:
Also just a comment: I just want to know whats going on here

Reply:
This is used to deserialize/serialize the model object.

Reply:
the schema class is used to map a database object fields to json encoding. the model class is the actual database object definition. nothing is being replaced here.